cranelift/isle/docs/language-reference.md

```diff
@@ -1458,45 +1460,43 @@ The grammar accepted by the parser is as follows:
 <ISLE> ::= <def>*
 
 <def> ::= "(" "pragma" <pragma> ")"
-        | "(" "type" <typedecl> ")"
+        | "(" "type" <type> ")"
```

Comment:
let's just use the dumbest mapping :)

Reply:
Eh, there is a reason I had chosen `typedecl` -- `type` is too close to `ty` (the latter reads as an abbreviation of the former with no semantic difference), the existing nonterminal for type expressions, and there is a semantic difference between uses of a type and the form that declares them. Let's stick with `typedecl`.